subsys/bluetooth/common/Kconfig

```diff
@@ -183,12 +183,44 @@ config BT_BUF_CMD_TX_SIZE
 	  to the maximum of 255.
 
 config BT_BUF_CMD_TX_COUNT
-	int "Number of HCI command buffers"
-	default 2
-	range 2 64
+	int "Number of HCI command buffers" if !BT_HCI_RAW || !HAS_BT_CTLR
+	# !BT_HCI_ACL_FLOW_CONTROL ==> Permit configurable Num_HCI_Command_Packets for
+	# Host-only build but hidden when HAS_BT_CTLR=y (default 1 for Controller-only build).
+	# !(BT_HCI_RAW && HAS_BT_CTLR) ==> Host-only build.
+	depends on !BT_HCI_RAW || !HAS_BT_CTLR || !BT_HCI_ACL_FLOW_CONTROL
```

Comment:
At least in my mind, it's easier to parse `!(A && B && C)` than `(!A || !B || !C)`. Maybe I'm alone in that, though :)

Reply:
I do actually prefer the latter :D That way you only, in your mind, have to parse a single value at a time :D Might be related to brain cache size :P

Reply:
Ok. Fair enough :)